sources/models/shared-defaults/defaults.toml

```diff
@@ -116,3 +116,12 @@ restart-commands = ["/usr/bin/bootstrap-containers create-containers"]
 
 [metadata.settings.bootstrap-containers]
 affected-services = ["bootstrap-containers"]
+
+# Certdog
```

Comment:
Are there services that need to be restarted when the bundle changes? Or does everything read the bundle 'just in time'?

Reply:
I think everything access to the bundle at runtime. In my test, I didn't have to restart `containerd` when I updated the bundle.